<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ALT Packages</string>
  </property>
  <widget class="PackagesWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="viewMenu">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="languageMenu">
     <property name="title">
      <string>&amp;Language</string>
     </property>
    </widget>
    <addaction name="languageMenu"/>
   </widget>
   <widget class="QMenu" name="helpMenu">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="manualAction"/>
    <addaction name="aboutAction"/>
   </widget>
   <widget class="QMenu" name="toolsMenu">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <widget class="QMenu" name="systemMenu">
     <property name="title">
      <string>&amp;System</string>
     </property>
     <addaction name="disableRemoveManuallyAction"/>
    </widget>
    <addaction name="systemMenu"/>
   </widget>
   <addaction name="viewMenu"/>
   <addaction name="helpMenu"/>
   <addaction name="toolsMenu"/>
  </widget>
  <action name="manualAction">
   <property name="text">
    <string>&amp;Manual</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="disableRemoveManuallyAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Disable the removal of packages installed manually</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PackagesWidget</class>
   <extends>QWidget</extends>
   <header location="global">ui/mainwindow/packageswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
